src/neynar-api/neynar-api-client.ts

```diff
@@ -2444,24 +2450,28 @@ export class NeynarAPIClient {
   // ------------ Frame ------------
 
   /**
-   * Retrieves a frame by its UUID, provided it was created by the developer identified by the provided API key.
+   * Retrieves a frame by its UUID or URL, provided it was created by the developer identified by the provided API key.
    * This method is useful for fetching details of a specific frame for review or display purposes.
    *
-   * @param {string} uuid - The UUID of the frame to be retrieved.
+   * @param {string} identifier - The UUID of the frame to be retrieved.
+   * @param {Object} [options] - Optional parameters for customizing the response.
+   * @param {FrameType} [options.type] - The type of identifier being used to query the frame.
```

Comment:
We'll need to add additional description for the type i.e. something like (for URL identifier add type=FrameType.Url)